docs/BR.md

```diff
@@ -931,9 +952,10 @@ Confirming the Applicant's control over the IP Address by performing the procedu
 
 #### 3.2.2.6 Wildcard Domain Validation
 
-Before issuing a certificate with a wildcard character (\*) in a CN or `subjectAltName` of type DNS-ID, the CA MUST establish and follow a documented procedure that determines if the wildcard character occurs in the first label position to the left of a "registry-controlled" label or "public suffix" (e.g. "\*.com", "\*.co.uk", see RFC 6454 Section 8.2 for further explanation).
+Before issuing a Wildcard Certificate, the CA MUST establish and follow a documented procedure that determines if the FQDN portion of any
+Wildcard Domain Name in the Certificate is “registry-controlled” or is a “public suffix” (e.g. “\*.com”, “\*.co.uk”, see RFC 6454 Section 8.2 for further explanation).
 
-If a wildcard would fall within the label immediately to the left of a registry-controlled or public suffix, CAs MUST refuse issuance unless the applicant proves its rightful control of the entire Domain Namespace. (e.g. CAs MUST NOT issue "\*.co.uk" or "\*.local", but MAY issue "\*.example.com" to Example Co.).
+If the FQDN portion of any Wildcard Domain Name is "registry-controlled" or is a "public suffix", CAs MUST refuse issuance unless the Applicant proves its rightful control of the entire Domain Namespace. (e.g. CAs MUST NOT issue "\*.co.uk" or "\*.local", but MAY issue "\*.example.com" to Example Co.).
```

Comment:
Minor nit: there is an inconsistency here between smart quotes (line 956: `“registry-controlled”`) and straight quotes (line 958: `"registry-controlled"`).

Reply:
Unless others disagree, it appears that the BRs use both types of quoting (straight vs. smart) so I think we should decide on which type of quotes we want in the documents and then fix all occurrences at once. Given that the documents are already inconsistent and mixed usage doesn't impair readability, I don't think we should delay the ballot further to address instance of mixed use.

Reply:
Yeah, we can leave these as-is for this ballot. Pandoc will automatically convert these to ligatures for the PDF rendering, so it makes no difference.

Reply:
Agreed, no need to delay this ballot to fix it. I only brought it up because both of these (mismatched) quotes were introduced in this ballot, not just a simple matter of this ballot using one style adjacent to where the other style already existed.